Libplanet.Explorer.Executable/Program.cs

```diff
@@ -30,11 +29,135 @@ namespace Libplanet.Explorer.Executable
+            [Option(
+                "block-interval",
+                new[] { 'i' },
+                Description = @"An appropriate interval in milliseconds between
+consecutive blocks.")]
+            int blockIntervalMilliseconds = 5000,
+            [Option(
+                "minimum-difficulty",
+                new[] { 'm' },
+                Description = "Allowed minimum difficulty for mining blocks.")]
+            long minimumDifficulty = 1024L,
+            [Option(
+                "difficulty-bound-divisor",
+                new[] { 'D' },
+                Description = "A bound divisor to determine precision of block difficulties.")]
+            int difficultyBoundDivisor = 128,
+            [Option(
+                "workers",
+                new[] { 'W' },
+                Description = "The number of swarm workers.")]
+            int workers = 50,
+            [Option(
+                "app-protocol-version",
+                new[] { 'V' },
+                Description = "An app protocol version token.")]
+            string appProtocolVersionToken = null,
+            [Option(
+                "mysql-server",
+                Description = "A hostname of MySQL server.")]
+            string mysqlServer = null,
+            [Option(
+                "mysql-port",
+                Description = "A port of MySQL server.")]
+            uint? mysqlPort = null,
+            [Option(
+                "mysql-username",
+                Description = "The name of MySQL user.")]
+            string mysqlUsername = null,
+            [Option(
+                "mysql-password",
+                Description = "The password of MySQL user.")]
+            string mysqlPassword = null,
+            [Option(
+                "mysql-database",
+                Description = "The name of MySQL database to use.")]
+            string mysqlDatabase = null,
+            [Option(
+                "max-transactions-per-block",
+                Description = @"The number of maximum transactions able to be included
+in a block.")]
+            int maxTransactionsPerBlock = 100,
+            [Option(
+                "max-block-bytes",
+                Description = @"The number of maximum bytes size of blocks except
+for genesis block.")]
+            int maxBlockBytes = 100 * 1024,
+            [Option(
+                "max-genesis-bytes",
+                Description = "The number of maximum bytes size of the genesis block.")]
+            int maxGenesisBytes = 1024 * 1024,
+            [Option(
+                "seed",
+                new[] { 's' },
+                Description = @"Seed nodes to join to the network as a node. The format of each
+seed is a comma-separated triple of a peer's hexadecimal public key, host, and port number.
+E.g., `02ed49dbe0f2c34d9dff8335d6dd9097f7a3ef17dfb5f048382eebc7f451a50aa1,example.com,31234'.
+If omitted (default) explorer only the local blockchain store.")]
+            string[] seedStrings = null,
```

Comment:
When I test this in my local, `seedStrings` is optional and it occurs `System.ArgumentNullException: Value cannot be null.` at the setter of `Options.SeedStrings`. Could you handle this?

Reply:
Thanks, I'll look into this part!